### PR TITLE
docker-images: add filter example

### PR DESCRIPTION
**Pull request by alimalek71**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/docker-images.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - List all Docker images not used by any container:
 
 `docker images --filter dangling=true`
+
+- List add Docker images contain phrase:
```
Comment (navarroaxel) [inline pages/common/docker-images.md:22]: Based on the `docker-ps.md` page:
```suggestion
- List images that contain a substring in their name:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @alimalek71. I left some comments.

Could you please edit the other examples, replacing `-a` by `--all` and `-q` for `--quiet`?

**Comment by alimalek71** [comment]
> Thanks @alimalek71. I left some comments.
> 
> Could you please edit the other examples, replacing `-a` by `--all` and `-q` for `--quiet`?

Done.

File: pages/common/docker-images.md [hunk 2]
```diff
@@ -9,12 +9,16 @@
...
-`docker images -q`
+`docker images --quiet`
 
 - List all Docker images not used by any container:
 
 `docker images --filter dangling=true`
+
+- List images that contain a substring in their name:
+
+`docker images {{"*name*"}}`
```
Comment (navarroaxel) [inline pages/common/docker-images.md:24]: Out standard is quotes outside the `{{}}`:
```suggestion
`docker images "{{*name*}}"`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @alimalek71 :slightly_smiling_face:

**Pull request merged**